src/coreclr/inc/corinfo.h

```diff
@@ -2853,6 +2888,7 @@ class ICorStaticInfo
         /* OUT */   SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR* structPassInRegDescPtr
         ) = 0;
 
+    virtual uint32_t getLoongArch64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE cls) = 0;
```

Comment:
> This needs a comment stating what the `uint32_t` return type returns.
> 
> If the argument type must be a struct handle, maybe use `structHnd` instead of `cls`, as `getSystemVAmd64PassStructInRegisterDescriptor` does?

OK, Thanks for guidance.

> 
> Is the return actually `StructFloatFieldInfoFlags`? If so, can you just use `StructFloatFieldInfoFlags` instead of `uint32_t`?

Here using `uint32_t` instead of `StructFloatFieldInfoFlags` is for composition of different `StructFloatFieldInfoFlags`.